net-libs/libident/libident-0.32-r1.ebuild

```diff
@@ -10,12 +12,19 @@ SRC_URI="http://people.via.ecp.fr/~rem/libident/${P}.tar.bz2"
 LICENSE="public-domain"
 SLOT="0"
 KEYWORDS="~alpha amd64 ~arm arm64 ~hppa ~ia64 ~mips ppc ppc64 ~s390 ~sparc x86"
-IUSE=""
 
-RDEPEND=""
-DEPEND="${RDEPEND}"
+DOCS=( AUTHORS ChangeLog NEWS README )
```

Comment:
why did you add this one? I'm assuming it installs something like `README.BeOS`

Reply:
To drop `dodoc` from `src_install`, I can return `dodoc` if you wish